Makefile

```diff
@@ -169,10 +168,10 @@ COCKROACH := ./cockroach$(SUFFIX)$(shell $(XGO) env GOEXE)
 .DEFAULT_GOAL := all
 all: $(COCKROACH)
 
-buildoss: BUILDTARGET = ./pkg/cmd/cockroach-oss
 buildoss: $(C_LIBS_OSS)
 
 $(COCKROACH) build go-install gotestdashi generate lint lintshort: $(C_LIBS_CCL)
+$(COCKROACH) build go-install gotestdashi generate lint lintshort: TAGS += ccl
```

Comment:
Maybe a first pass can be to just add this line, and leave the rest as-is.  Would that be acceptable to folks, or is introducing this at all a concern?

I'm generally a fan of having just one way to do things, so if there's a build tag for CCL, that seems like the way all the CCL code should get built in.  Going the other way is harder: it's not immediately clear how to nicely swap out which `embedded.go` to use from the top-level entry point.

Reply:
Yeah, if you want to sidecar this line into your PR, @couchand, I'll just file this as future cleanup work.